web/modules/custom/simplytest_tugboat/src/PreviewConfigGenerator.php

```diff
@@ -132,7 +132,7 @@ public function oneClickDemo(string $demo_id, array $parameters): array {
       ['echo "SIMPLYEST_STAGE_PATCHING"'],
       $one_click_demo->getPatchingCommands($parameters),
       [
-        'cd stm && composer update --no-ansi',
+        'cd stm && composer config allow-plugins true && composer update --no-ansi',
```

Comment:
We don't need to do this here, it's a bug due to the legacy Drupal Commerce project template.